docs/UserGuide.md

```diff
@@ -38,97 +38,204 @@
+
+> If DATE is entered but TIME is not entered, the task is assumed to occur all-day <br>
+> If TIME is entered but DATE is not entered, DATE is assumed to be current date <br>
+> Tasks with START_TIME only, are assumed to occur for 1 hour from specified START_TIME only <br>
+> Tasks with either one of or both END_DATE and/or END_TIME, are assumed to be tasks with deadlines <br>
+> Tasks with START_DATE only, are assumed to have the same START_DATE and END_DATE <br>
+> Tasks with either or both START_DATE and/or START_TIME, are assumed to be events instead of task <br>
+> If START_DATE and END_DATE are different, the RECURRING_TYPE has to be larger than the duration between START_DATE and END_DATE. (e.g. A 3d2n camp cannot be recurring daily but can be recurring weekly) <br>
+> If START_DATE and END_DATE are different, END_DATE must be later than START_DATE, and END_TIME does not have to be later than START_TIME <br>
+> If START_DATE and END_DATE are the same, END_TIME must be later than START_TIME <br>
+> For DATE the format is as follows: dd-mm-yyyy <br>
+> For TIME the format is as follows: hh:MM, hh:MM defaults to 00:00 if not specified. <br>
+> Having more than 1 task for any specific timing is allowed (Multitasking) <br>
+> RECURRING_TYPE can be `none`, `daily`, `weekly` or `monthly`, `yearly`, by default RECURRING_TYPE set as `none`, NUMBER_OF_RECURRENCE set as 0 <br>
+> PRIORITY_LEVEL can be `high`, `medium`, `low`, by default PRIORITY_LEVEL is set as `low` <br>
 
-> Persons can have any number of tags (including 0)
+Examples: 
+* `add Project Meeting s/05-10-2016 st/14:00 et/18:00 r/daily n/2 c/CS2103 d/Discuss about roles and milestones` <br>
+  Add task named, Project Meeting, under CS2103 category. The task is schedule to take place on 5th and 6th of October 2016 from 2pm to 6pm each day.
+* `add NUSSU Leadership Camp s/05-10-2016 st/14:00 e/08-10-2016 et/18:00 c/NUSSU`
+  Add task named, NUSSU Leadership Camp, under NUSSU category. The 4 day 3 night is schedule to take place from 5th October, 2pm to 8th of October 2016, 6pm.
+
+#### Listing all tasks: `list`
+Shows a list of all tasks in Savvy Tasker <br>
+Format: `list [t/LIST_TYPE]`
+
+> LIST_TYPE can be `Due Date`, `Priority Level`, `Archived`, by default LIST_TYPE is set as `Due Date`<br>
+> If TYPE is `Due Date`, the tasks and events are sorted according to due date and time of tasks and start date and time of events, earliest first.<br>
+> If no ENDTIME specified (floating tasks), sorted to bottom of list.<br>
+> If TYPE is `Priority Level`, the tasks and events are sorted according to priority level, high first.<br>
+> If TYPE is `Archived`, the Archived tasks and events are listed. They are sorted according to date the task has been marked, most recent first.<br>
+
+#### Finding all task containing any keyword in its name: `find`
+Finds tasks whose names contain any of the given keywords.<br>
+Format: `find [t/FIND_TYPE] KEYWORD [MORE_KEYWORDS]`
```

Comment:
the find type can be a low priority item. since you default to Partial search, it is good enough for starters.
i see you have many other useful things you are providing, so this can be pushed back in the schedule, if you wish!